merlin/models/tf/outputs/topk.py

```diff
@@ -100,7 +100,7 @@ def extract_ids_embeddings(self, data: merlin.io.Dataset, check_unique_ids: bool
         if check_unique_ids:
             self._check_unique_ids(data=data)
         values = tf_utils.df_to_tensor(data)
-        ids = tf_utils.df_to_tensor(data.index)
+        ids = tf_utils.df_to_tensor(make_df({"index": data.index}))
```

Comment:
This breaks with the newer versions of cudf. A RangeIndex no longer has an `empty` property and won't work with this df_to_tensor function.